custom_components/bhyve/sensor.py

```diff
@@ -244,6 +244,7 @@ def __init__(self, hass, bhyve, device):
     def _setup(self, device):
         self._state = device.get("status", {}).get("temp_f")
         self._available = device.get("is_connected", False)
+        self._unit = "°F"
```

Comment:
Please replace with `TEMP_FAHRENHEIT`, which can be imported from `homeassistant.const`